allennlp/data/fields/labels_field.py

```diff
@@ -0,0 +1,135 @@
+from typing import Dict, List, Union, Set, Iterator
+import logging
+import textwrap
+
+from overrides import overrides
+import torch
+
+from allennlp.common.checks import ConfigurationError
+from allennlp.common.util import pad_sequence_to_length
+from allennlp.data.fields.sequence_field import SequenceField
+from allennlp.data.vocabulary import Vocabulary
+
+logger = logging.getLogger(__name__)
+
+
+class LabelsField(SequenceField):
```

Comment:
What does this provide on top of our [`SequenceLabelField`](https://github.com/allenai/allennlp/blob/master/allennlp/data/fields/sequence_label_field.py)?